perception/ground_segmentation/src/scan_ground_filter_nodelet.cpp

```diff
@@ -285,10 +287,13 @@ void ScanGroundFilterComponent::recheckGroundCluster(
 }
 void ScanGroundFilterComponent::classifyPointCloudGridScan(
   std::vector<PointCloudRefVector> & in_radial_ordered_clouds,
-  pcl::PointIndices & out_no_ground_indices)
+  std::vector<pcl::PointIndices> & out_no_ground_indices)
 {
-  out_no_ground_indices.indices.clear();
+  out_no_ground_indices.resize(in_radial_ordered_clouds.size());
+#pragma omp parallel for
```

Comment:
I think it's better to set the number of threads 
```suggestion
  omp_set_num_threads(omp_num_threads_);
#pragma omp parallel for
```

where 

```
  omp_num_threads_ = declare_parameter<int>("omp_num_threads");
```